gaia-list.js

```diff
@@ -82,15 +85,20 @@ module.exports = component.register('gaia-list', {
     this.els.inner.appendChild(els.container);
 
     // var reflow = els.ripple.offsetTop;
-    var scale = pos.item.width / 1.2;
+    var scale = itemWidth / 1.2;
+    // tap at edge of the list
```

Comment:
We need more description here about about what the intended behaviour is and how we are achieving it.